packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -2103,43 +2169,46 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
       child: NotificationListener<ScrollNotification>(
         onNotification: _handleScrollNotification,
         child: RepaintBoundary(
-          child: RawGestureDetector(
-            gestures: _gestures,
-            child: MouseRegion(
-              onExit: (PointerExitEvent event) {
-                switch(event.kind) {
-                  case PointerDeviceKind.mouse:
-                  case PointerDeviceKind.trackpad:
-                    if (enableGestures) {
-                      handleHoverExit(event);
-                    }
-                    break;
-                  case PointerDeviceKind.stylus:
-                  case PointerDeviceKind.invertedStylus:
-                  case PointerDeviceKind.unknown:
-                  case PointerDeviceKind.touch:
-                    break;
-                }
-              },
-              onHover: (PointerHoverEvent event) {
-                switch(event.kind) {
-                  case PointerDeviceKind.mouse:
-                  case PointerDeviceKind.trackpad:
-                    if (enableGestures) {
-                      handleHover(event);
-                    }
-                    break;
-                  case PointerDeviceKind.stylus:
-                  case PointerDeviceKind.invertedStylus:
-                  case PointerDeviceKind.unknown:
-                  case PointerDeviceKind.touch:
-                    break;
-                }
-              },
-              child: CustomPaint(
-                key: _scrollbarPainterKey,
-                foregroundPainter: scrollbarPainter,
-                child: RepaintBoundary(child: widget.child),
+          child: Listener(
```

Comment:
So this took me a minute because I thought, why can't we approach this through hit testing? Over course you already had an answer for me: https://github.com/flutter/flutter/pull/99328 😜 

I don't love having so many handlers and listeners here, like MouseRegion (for hovering), Listener (for pointer scrolling), NotificationListeners (for scrolling and scroll metrics),  but they do seem necessary to be able to match all of the various behaviors we see on different platforms.